src/tv2_afvd_showstyle/__tests__/blueprint.spec.ts

```diff
@@ -729,9 +729,10 @@ describe('AFVD Blueprint', () => {
 	})
 
 	it('Changes design and background loops', async () => {
+		// Changed to use the made up "DESIGN_FODBOLD_22" so as to avoid finding a non-transferred design in GFXTemplate
```

Comment:
Comments describing why stuff changed is allready covered by our commit messages. (One of the perks of using version control ;) )

A comment like this is going to become outdated really fast, better to not have it to begin with

Reply:
Removed in [91af7ee](https://github.com/tv2/sofie-blueprints-inews/pull/149/commits/91af7eea234989f636428c9e8cd8bd90a524675a)